package.json

```diff
@@ -119,7 +119,6 @@
     "better-sqlite3": "*",
     "hdb-pool": "*",
     "ioredis": "*",
-    "ioredis": "*",
```

Comment:
can you remove these changes

Reply:
Sure, this can be removed but seems like this is duplicate.